SQL/updates/5-6.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE  `ipintel` (
```

Comment:
Ideally a comment needs to go here, similar to the other SQL revisions

Reply:
Added